--- a/tools/xmlpatterns/xmlpatterns.pro	2012-11-10 19:03:24.000000000 +0000
+++ b/tools/xmlpatterns/xmlpatterns.pro	2012-12-05 05:29:25.372327814 +0000
@@ -1,5 +1,5 @@
 TEMPLATE = app
-TARGET = xmlpatterns
+TARGET = xmlpatterns-qt5
 QT -= gui
 QT += xmlpatterns-private
 
--- a/tools/xmlpatternsvalidator/xmlpatternsvalidator.pro	2012-11-10 19:03:24.000000000 +0000
+++ b/tools/xmlpatternsvalidator/xmlpatternsvalidator.pro	2012-12-05 05:29:15.340327499 +0000
@@ -1,5 +1,5 @@
 TEMPLATE = app
-TARGET = xmlpatternsvalidator
+TARGET = xmlpatternsvalidator-qt5
 QT -= gui
 QT += xmlpatterns
 
