Description: disable tests that require network connection
Author: Iftikhar Ahmad <iftikhar.ahmad@canonical.com>
Forwarded: not-needed
Last-Update: 2014-02-10

--- a/tests/auto/auto.pro
+++ b/tests/auto/auto.pro
@@ -19,10 +19,8 @@ SUBDIRS=\
            qxmlname                     \
            qxmlnamepool                 \
            qxmlnodemodelindex           \
-           qxmlquery                    \
            qxmlresultitems              \
            qxmlserializer               \
-           xmlpatterns                  \
            xmlpatternsdiagnosticsts     \
            xmlpatternssdk               \
            xmlpatternsvalidator         \
@@ -30,6 +28,10 @@ SUBDIRS=\
            xmlpatternsxqts              \
            xmlpatternsxslts             \
 
+#skip the network tests for unix platform.
+!unix:SUBDIRS += qxmlquery
+!unix:SUBDIRS += xmlpatterns
+
 load(qfeatures)
 !contains(QT_DISABLED_FEATURES, xmlschema) {
     SUBDIRS += qxmlschema               \
